x-pack/test/accessibility/apps/ml.ts

```diff
@@ -10,18 +10,21 @@ import { FtrProviderContext } from '../ftr_provider_context';
 export default function ({ getService }: FtrProviderContext) {
   const a11y = getService('a11y');
   const ml = getService('ml');
+  const kibanaServer = getService('kibanaServer');
 
   describe('ml Accessibility', () => {
     const esArchiver = getService('esArchiver');
 
     before(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
```

Comment:
Why are we doing this here?  I ask as I dont see any removals of using empty_kibana.

Reply:
AFAICS, the test failure that was supposed to be fixed with this [change](25e5db3d597296b046565dff060f066a2b0dedae) was actually around a left over space and was fixed in a different [commit](8b3ed695d5cbd51a765403866cc3b6569d3e2416). As a result, we don't really need these change in the ml file, but it also doesn't hurt to ensure a clean environment. I'm fine with or without these changes.

Reply:
Thanks @wayneseymour @pheyos I have removed it. This is also green in flaky test runner. Cheers.